Resources/Prototypes/Entities/Mobs/NPCs/dogs.yml

```diff
@@ -69,13 +69,7 @@
   - type: DogVision
   - type: NpcFactionMember
     factions:
-    - Pibble
-  - type: HTN
-    rootTask:
-      task: SimpleHostileCompound
-    blackboard:
-      NavSmash: !type:Bool
-        true
```

Comment:
Don't remove lines, simply comment them out.

Reply:
You right